strategies/builderbid/best/builderbid.go

```diff
@@ -243,7 +243,12 @@ func (*Service) setBuilderBid(ctx context.Context,
 		score = new(big.Int).Add(score, builderConfig.Offset)
 	}
 	if builderConfig.Factor != nil {
-		score = new(big.Int).Mul(score, builderConfig.Factor)
+		if builderConfig.Factor.Cmp(big.NewInt(0)) > 0 {
```

Comment:
This seems overly complex.  What about `score = new(big.Int).Div(new(big.Int).Mul(score, builderConfig.Factor), big.NewInt(100))`?

Note that if we want to disallow factors <0 then we should do so when reading in the configuration rather than here.

Reply:
(Note that another PR has addressed that factors cannot be <0)